web/src/containers/LoanRequest.js

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {
+  TextField,
+  Button,
+  Grid,
+  Typography,
+  withStyles,
+  InputAdornment,
+} from "@material-ui/core";
+import { Alert } from "@material-ui/lab";
+import { useFormik } from "formik";
+import * as Yup from "yup";
+import LoanRequestService from "../services/loanrequest.service";
+import { useHistory } from "react-router";
+
+const styles = {
```

Comment:
We are once again repeating this block of CSS. Shouldn't this be moved to the layout?